automatic/firefox/README.md

```diff
@@ -17,7 +17,22 @@ Bringing together all kinds of awesomeness to make browsing better for you.
 
 ## Package Parameters
 
-- `l=<locale>` - Install given Firefox locale. For example `choco install Firefox --params "l=en-GB"`. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+- `l=<locale>` - Install given Firefox locale. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+
+Command-line options for installer configuration. See the [official page](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html) for details and defaults.
+- `/InstallDirectoryPath=<path>`
+- `/InstallDirectoryName=<path>`
+- `/TaskbarShortcut=<true/false>`
+- `/DesktopShortcut=<true/false>`
+- `/StartMenuShortcut=<true/false>`
+- `/MaintenanceService=<true/false>`
```

Comment:
I would remove this param and always specify it, it goes along with unatended spirit of choco.

Reply:
I read your comment as to not install the Maintenance Service by default. Is that correct? Currently Firefox updates itself when installed through chocolatey. I wouldn't want to change that because I assume many people use chocolatey as "one-time installer" and the rely on the update mechanism off FF.

Reply:
I didn't understand it that way. If it is related to autoupdate, then lots of people would actually like to disable it and use choco instead. So `/NoAutoUpdate` would be nice option for those people. 

I understood that such FF option does that actually. Take a look:

> This service allows Firefox to install updates without requiring you to click yes in the Windows User Account Control (UAC) dialog - Read more at https://www.shouldiremoveit.com/Mozilla-Maintenance-Service-5231-program.aspx

So it looks like it prevents UAC prompt only by runing maintenance service as Windows service. Without it, you still have maintenance service but it is not Windows service, and you have to click. If that is so, that FF option should be alwasy specified (why would I click in choco land?).

Now, if you find a way to implement `/NoAutoUpdate` even better. Such option is HIGHLY needed for developers using Selenium for example as they always need to prevent automatic updates.

Reply:
You're right, i mixed that up. I like the idea with `/NoAutoUpdate`. If set the MaintenanceService should not be installed. The auto update of FF can be disabled by setting `policies.json`
```
{
  "policies": {
    "DisableAppUpdate": true
  }
}
```